RetroArch 1.8.9 not working on windows 10

## Description

[Description of the bug]

RetroArch 1.8.9 Nightly and Stable not working on windows 10


### Actual behavior

retroarch_debug.exe --verbose 
retroarch.cfg
`video_driver = "gl"`
```
[INFO] RetroArch 1.8.9 (Git 6fcd1406c3)
[INFO] === Build =======================================
[INFO] CPU Model Name: Intel(R) Core(TM) i7-3770 CPU @ 3.40GHz
[INFO] 容量:  MMX MMXEXT SSE SSE2 SSE3 SSSE3 SSE4 SSE4.2 AES AVX
[INFO] Built: Jul  3 2020
[INFO] Version: 1.8.9
[INFO] Git: 6fcd1406c3
[INFO] =================================================
[INFO] Initializing Dr.MingW Exception handler
[INFO] [Environ]: SET_PIXEL_FORMAT: RGB565.
[INFO] [Overrides]: 重定向游戏存档文件至 "I:\WinTemp\UsrTmp\RetroArch-1\saves\.srm".
[INFO] [Overrides]: 正在重定向状态存储至 "I:\WinTemp\UsrTmp\RetroArch-1\states\.state".
[INFO] libretro API 版本: 1
[INFO] 未按照 API 编译: 1
[INFO] [Audio]: Set audio input rate to: 48000.00 Hz.
[INFO] [Video]: Video @ 960x720
[ERROR] [D3D12]: win32_set_video_mode failed.
[ERROR] [D3D12]: failed to init video driver.
[ERROR] [Video]: Cannot open video driver ... Exiting ...
[ERROR] Fatal error received in: "init_video()"
```
retroarch.cfg
`video_driver = "d3d9"`
```
[INFO] RetroArch 1.8.9 (Git 6fcd1406c3)
[INFO] === Build =======================================
[INFO] CPU Model Name: Intel(R) Core(TM) i7-3770 CPU @ 3.40GHz
[INFO] 容量:  MMX MMXEXT SSE SSE2 SSE3 SSSE3 SSE4 SSE4.2 AES AVX
[INFO] Built: Jul  3 2020
[INFO] Version: 1.8.9
[INFO] Git: 6fcd1406c3
[INFO] =================================================
[INFO] Initializing Dr.MingW Exception handler
[INFO] [Environ]: SET_PIXEL_FORMAT: RGB565.
[INFO] [Overrides]: 重定向游戏存档文件至 "I:\WinTemp\UsrTmp\RetroArch-1\saves\.srm".
[INFO] [Overrides]: 正在重定向状态存储至 "I:\WinTemp\UsrTmp\RetroArch-1\states\.state".
[INFO] libretro API 版本: 1
[INFO] 未按照 API 编译: 1
[INFO] [Audio]: Set audio input rate to: 48000.00 Hz.
[INFO] [Video]: Video @ 960x720
[INFO] [D3D9]: Monitor size: 1920x1080.
[ERROR] [D3D9]: Failed to initialize device.
[ERROR] [D3D9]: Failed to init D3D.
[ERROR] [Video]: Cannot open video driver ... Exiting ...
[ERROR] Fatal error received in: "init_video()"
```
retroarch.cfg
`video_driver = "d3d10"`
```
[INFO] RetroArch 1.8.9 (Git 6fcd1406c3)
[INFO] === Build =======================================
[INFO] CPU Model Name: Intel(R) Core(TM) i7-3770 CPU @ 3.40GHz
[INFO] 容量:  MMX MMXEXT SSE SSE2 SSE3 SSSE3 SSE4 SSE4.2 AES AVX
[INFO] Built: Jul  3 2020
[INFO] Version: 1.8.9
[INFO] Git: 6fcd1406c3
[INFO] =================================================
[INFO] Initializing Dr.MingW Exception handler
[INFO] [Environ]: SET_PIXEL_FORMAT: RGB565.
[INFO] [Overrides]: 重定向游戏存档文件至 "I:\WinTemp\UsrTmp\RetroArch-1\saves\.srm".
[INFO] [Overrides]: 正在重定向状态存储至 "I:\WinTemp\UsrTmp\RetroArch-1\states\.state".
[INFO] libretro API 版本: 1
[INFO] 未按照 API 编译: 1
[INFO] [Audio]: Set audio input rate to: 48000.00 Hz.
[INFO] [Video]: Video @ 960x720
[ERROR] [D3D10]: win32_set_video_mode failed.
[ERROR] [Video]: Cannot open video driver ... Exiting ...
[ERROR] Fatal error received in: "init_video()"
```
retroarch.cfg
`video_driver = "d3d11"`
```
[INFO] RetroArch 1.8.9 (Git 6fcd1406c3)
[INFO] === Build =======================================
[INFO] CPU Model Name: Intel(R) Core(TM) i7-3770 CPU @ 3.40GHz
[INFO] 容量:  MMX MMXEXT SSE SSE2 SSE3 SSSE3 SSE4 SSE4.2 AES AVX
[INFO] Built: Jul  3 2020
[INFO] Version: 1.8.9
[INFO] Git: 6fcd1406c3
[INFO] =================================================
[INFO] Initializing Dr.MingW Exception handler
[INFO] [Environ]: SET_PIXEL_FORMAT: RGB565.
[INFO] [Overrides]: 重定向游戏存档文件至 "I:\WinTemp\UsrTmp\RetroArch-1\saves\.srm".
[INFO] [Overrides]: 正在重定向状态存储至 "I:\WinTemp\UsrTmp\RetroArch-1\states\.state".
[INFO] libretro API 版本: 1
[INFO] 未按照 API 编译: 1
[INFO] [Audio]: Set audio input rate to: 48000.00 Hz.
[INFO] [Video]: Video @ 960x720
[ERROR] [D3D11]: win32_set_video_mode failed.
[INFO] [config] 已保存新配置到 "I:\WinTemp\UsrTmp\RetroArch-1\retroarch.cfg".
```
retroarch.cfg
`video_driver = "d3d12"`
```
[INFO] RetroArch 1.8.9 (Git 6fcd1406c3)
[INFO] === Build =======================================
[INFO] CPU Model Name: Intel(R) Core(TM) i7-3770 CPU @ 3.40GHz
[INFO] 容量:  MMX MMXEXT SSE SSE2 SSE3 SSSE3 SSE4 SSE4.2 AES AVX
[INFO] Built: Jul  3 2020
[INFO] Version: 1.8.9
[INFO] Git: 6fcd1406c3
[INFO] =================================================
[INFO] Initializing Dr.MingW Exception handler
[INFO] [Environ]: SET_PIXEL_FORMAT: RGB565.
[INFO] [Overrides]: 重定向游戏存档文件至 "I:\WinTemp\UsrTmp\RetroArch-1\saves\.srm".
[INFO] [Overrides]: 正在重定向状态存储至 "I:\WinTemp\UsrTmp\RetroArch-1\states\.state".
[INFO] libretro API 版本: 1
[INFO] 未按照 API 编译: 1
[INFO] [Audio]: Set audio input rate to: 48000.00 Hz.
[INFO] [Video]: Video @ 960x720
[ERROR] [D3D12]: win32_set_video_mode failed.
[ERROR] [D3D12]: failed to init video driver.
[ERROR] [Video]: Cannot open video driver ... Exiting 
[ERROR] Fatal error received in: "init_video()"
```
retroarch.cfg
`video_driver = "vulkan"`
```
[INFO] RetroArch 1.8.9 (Git 6fcd1406c3)
[INFO] === Build =======================================
[INFO] CPU Model Name: Intel(R) Core(TM) i7-3770 CPU @ 3.40GHz
[INFO] 容量:  MMX MMXEXT SSE SSE2 SSE3 SSSE3 SSE4 SSE4.2 AES AVX
[INFO] Built: Jul  3 2020
[INFO] Version: 1.8.9
[INFO] Git: 6fcd1406c3
[INFO] =================================================
[INFO] Initializing Dr.MingW Exception handler
[INFO] [Environ]: SET_PIXEL_FORMAT: RGB565.
[INFO] [Overrides]: 重定向游戏存档文件至 "I:\WinTemp\UsrTmp\RetroArch-1\saves\.srm".
[INFO] [Overrides]: 正在重定向状态存储至 "I:\WinTemp\UsrTmp\RetroArch-1\states\.state".
[INFO] libretro API 版本: 1
[INFO] 未按照 API 编译: 1
[INFO] [Audio]: Set audio input rate to: 48000.00 Hz.
[INFO] [Video]: Video @ 960x720
[INFO] [Vulkan]: Vulkan dynamic library loaded.
[INFO] [Vulkan]: Found vulkan context: wgl
[INFO] [Vulkan]: Detecting screen resolution 1920x1080.
[ERROR] [WGL]: win32_set_video_mode failed.
[ERROR] [Vulkan]: Failed to set video mode.
[ERROR] [Video]: Cannot open video driver ... Exiting ...
[ERROR] Fatal error received in: "init_video()"
```

### Steps to reproduce the bug

only run RetroArch 1.8.9

### Bisect Results

[Try to bisect and tell us when this started happening]

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.8.9 Git 6fcd1406c3 (Nightly) and Git 4c6b13919c (Stable)
- Last Woring RetroArch: 1.8.8 Git 02bc3ee1e8

### Environment information

- OS:Windows 10 with 2004 
- GPU:GTX 1070 TI with 451.48 dch driver
- Compiler: none

I had the same problem.

_[INFO] RetroArch 1.8.9 (Git 4c6b13919c)
[INFO] === Build =======================================
[INFO] CPU Model Name: Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz
[INFO] 容量:  MMX MMXEXT SSE SSE2 SSE3 SSSE3 SSE4 SSE4.2 AES AVX
[INFO] Built: Jun 19 2020
[INFO] Version: 1.8.9
[INFO] Git: 4c6b13919c
[INFO] =================================================
[INFO] [Environ]: SET_PIXEL_FORMAT: RGB565.
[INFO] [Overrides]: 重定向游戏存档文件至 "E:\RetroArch\saves\.srm".
[INFO] [Overrides]: 正在重定向状态存储至 "E:\RetroArch\states\.state".
[INFO] libretro API 版本: 1
[INFO] 未按照 API 编译: 1
[INFO] [Audio]: Set audio input rate to: 48000.00 Hz.
[INFO] [Video]: Video @ 960x720
[INFO] [GL]: Found GL context: wgl
[INFO] [GL]: Detecting screen resolution 1920x1080.
[ERROR] [WGL]: win32_set_video_mode failed.
[ERROR] [Video]: Cannot open video driver ... Exiting ...
[ERROR] Fatal error received in: "init_video()"_

I tried all methods in #4580, but all of them don't work.

--- a/gfx/common/win32_common.c
+++ b/gfx/common/win32_common.c
@@ -1345,7 +1345,7 @@ bool win32_window_create(void *data, unsigned style,
       user_height                = g_win32->pos_height;
    }
    main_window.hwnd              = CreateWindowEx(0,
-         msg_hash_to_str(MSG_PROGRAM), msg_hash_to_str(MSG_PROGRAM),
+         "RetroArch", msg_hash_to_str(MSG_PROGRAM),
          style,
          fullscreen ? mon_rect->left : g_win32->pos_x,
          fullscreen ? mon_rect->top  : g_win32->pos_y,
@@ -1820,7 +1820,7 @@ void win32_window_reset(void)
 void win32_destroy_window(void)
 {
 #ifndef _XBOX
-   UnregisterClass(msg_hash_to_str(MSG_PROGRAM), 
+   UnregisterClass("RetroArch", 
          GetModuleHandle(NULL));
 #if defined(_WIN32_WINNT) && _WIN32_WINNT >= 0x500 /* 2K */
    UnregisterDeviceNotification(notification_handler);

--- a/ui/drivers/ui_win32.c
+++ b/ui/drivers/ui_win32.c
@@ -70,7 +70,7 @@ bool win32_window_init(WNDCLASSEX *wndclass,
    wndclass->style         = CS_HREDRAW | CS_VREDRAW | CS_OWNDC;
    wndclass->hInstance     = GetModuleHandle(NULL);
    wndclass->hCursor       = LoadCursor(NULL, IDC_ARROW);
-   wndclass->lpszClassName = (class_name != NULL) ? class_name : msg_hash_to_str(MSG_PROGRAM);
+   wndclass->lpszClassName = (class_name != NULL) ? class_name : "RetroArch";
    wndclass->hIcon         = LoadIcon(GetModuleHandle(NULL), MAKEINTRESOURCE(IDI_ICON));
    wndclass->hIconSm       = (HICON)LoadImage(GetModuleHandle(NULL),
          MAKEINTRESOURCE(IDI_ICON), IMAGE_ICON, 16, 16, 0);

